pkgs/development/libraries/AntTweakBar/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, xlibs, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "AntTweakBar-1.16";
+
+  buildInputs = [ unzip xlibs.libX11 mesa ];
+
+  src = fetchurl {
+    url = "https://vorboss.dl.sourceforge.net/project/anttweakbar/AntTweakBar_116.zip";
```

Comment:
changed

Reply:
I ran into this issue: https://github.com/NixOS/nix/issues/969#issuecomment-362786187

Reply:
Oh well, the most fun case is with some tarballs where you can have a completely different name (but the same hash) and it will fetch from hash-addressed `tarballs.nixos.org`…